src/pages/learn.jsx

```diff
@@ -32,23 +34,38 @@ export const learn = () => {
         <li>Souther Resident Killer Whale Call Catalog</li>
         <li>Exhibits</li>
       </ul>
-      <h1 className={learnStyles.salish}> Sounds Of The Salish Sea</h1>
-      <p className={learnStyles.salishsea}>
-        Explore common sounds of the Salish Sea by selecting the animals and
-        other objects in this anoramic soundscape of the inlandwaters of
-        Washington State(USA) and British Columbia(Canada)
-      </p>
-      <div className={learnStyles.imageWrapper}>
+
+      <Box sx={{ marginLeft: '85px' }}>
+        <Typography variant="h1">
+          <h1 className={learnStyles.salish}> Sounds Of The Salish Sea</h1>
+        </Typography>
+      </Box>
+
+      <Box
+        width={969}
+        height={84}
+        sx={{ ml: '85px', marginTop: '30px', marginBottom: '29px' }}
+      >
+        <Typography variant="body1">
+          <p className={learnStyles.salishsea}>
```

Comment:
```suggestion
```
Same comment as above, we should just use sx prop and take this opportunity to remove some CSS module classes